packages/d/devextreme.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "devextreme",
+  "description": "HTML5 JavaScript Component Suite for Responsive Web Development",
+  "keywords": [
+    "html5",
+    "controls",
+    "widgets",
+    "components",
+    "jquery",
+    "knockout",
+    "angular",
+    "web",
+    "ui",
+    "datagrid",
+    "pivotgrid",
+    "scheduler",
+    "treelist",
+    "chart"
+  ],
+  "homepage": "https://js.devexpress.com/",
+  "license": "https://js.devexpress.com/Licensing/",
+  "authors": [
+    {
+      "name": "Developer Express Inc.",
+      "email": "management@devexpress.com",
+      "url": "https://www.devexpress.com/"
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/DevExpress/DevExtreme.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "devextreme",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "js/*.js",
```

```suggestion
          "js/!(*.debug).js",
```
Given the incredibly large size of the debug files, let's exclude those from being cloned.

[Glob](https://www.digitalocean.com/community/tools/glob?comments=true&glob=js%2F%21%28%2A.debug%29.js&matches=false&tests=js%2Fdx.viz-web.debug.js&tests=js%2Fdx.viz-web.js)